data/ sql/ updates/ pending_db_world/rev_1587689314000956500.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1587689314000956500');
+
+UPDATE `creature_template` SET `AIName`='SmartAI', `ScriptName`='' WHERE `entry`=25623;
```

Comment:
did the creature already have this before?

and is the "ScriptName" part useful?

Reply:
Yea, but when I learned how to do this stuff for the first time, you usually have it as a "just in case" but it is not neccessary